config/routes.rb

```diff
@@ -23,7 +23,9 @@
   #
 
   # Organization CRUD
-  resources :organizations, only: [:index, :create, :show, :update, :destroy], format: 'json'
+  resources :organizations, only: [:index, :create, :show, :update, :destroy], format: 'json' do
+    resources :events, only: [:index, :create, :show, :update, :destroy], format: 'json'
```

Comment:
Add standalone resources for `/events` to be able to add `/events/:event_id/attendees` without over nesting resources.